kubernetes/full-configs/k8s-guide/k8s_guide_config.yaml

```diff
@@ -113,9 +113,7 @@ resources:
         - blueprint: '"service"'
           icon: '"Deployment"'
           identifier: .metadata.labels.portService
-          properties: {}
-          relations:
-            prod_runtime: .metadata.name + "-Deployment-" + .metadata.namespace + "-" + "my-cluster"
           title: .metadata.name
+          properties: {}
```

Comment:
This is the only mapping block where properties is at the bottom, let's move it above `title` like in the rest of the examples